scripts/getDependencies.pl

```diff
@@ -94,6 +94,21 @@
 		fname => 'junit4.jar',
 		sha1 => 'e4f1766ce7404a08f45d859fb9c226fc9e41a861'
 	},
+	mockito_core => {
+		url => 'https://repo1.maven.org/maven2/org/mockito/mockito-core/5.9.0/mockito-core-5.9.0.jar',
+		fname => 'mockito-core.jar',
+		sha1 => 'faa88b96db3aeb96a93e83dec7491345bfbfc414'
+	},
+	byte_buddy => {
+		url => 'https://repo1.maven.org/maven2/net/bytebuddy/byte-buddy/1.14.12/byte-buddy-1.14.12.jar',
+		fname => 'byte-buddy.jar',
+		sha1 => '6e37f743dc15a8d7a4feb3eb0025cbc612d5b9e1'
+	},
+	maven_jar_plugin => {
```

Comment:
@sophiaxu0424 My bad for missing this, I don't think we need this JAR file, can you submit another PR to remove this?